Directory.Packages.props

```diff
@@ -25,7 +24,7 @@
     <PackageVersion Include="Serilog.AspNetCore" Version="8.0.0" />
     <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.14.0.81108" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />
-    <PackageVersion Include="xunit" Version="2.6.4" />
+    <PackageVersion Include="xunit" Version="2.6.5" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.6" />

```

Comment:
**Suggestion:** The version of `xunit` has been updated to `2.6.5` while `xunit.runner.visualstudio` is still at `2.5.6`. If there is no specific reason for this, it would be better to keep them at the same version. [best practice]
```suggestion
    <PackageVersion Include="xunit" Version="2.6.5" />
    <PackageVersion Include="xunit.runner.visualstudio" Version="2.6.5" />
```